### PR TITLE
List group icon issue

### PR DESCRIPTION
**Pull request by h0rhay**
#### What does this PR do? (please provide any background)
https://github.com/holidayextras/brands/issues/122
http://www.holidayextras.co.uk/frequently-asked-questions.html

^^ Currently multiline list items are misaligned with icons.

This pushes and pulls the lis to line them up 👍 

#### How can this be tested?
`cd pattern-library && npm link ../brands`
Run SSG or Pattern Library locally and add `list-group-icon` to any ul that is a list with icons, eg: FAQ page on SSG..

#### Any tech debt?
nope
#### Screenshots / Screencast
Bad:
screen shot 2016-12-06 at 12 29 55

Good:
screen shot 2016-12-06 at 12 30 17

#### What gif best describes how you feel about this work?

- [ ] I have checked our general [contributing document](https://github.com/holidayextras/culture/blob/master/CONTRIBUTING.md) and the project specific [contributing document](../blob/master/CONTRIBUTING.md) (if present) and I'm happy for this to be reviewed.

---

**Reviewer**
- [ ] :+1:
- [ ] UXUI Team :+1:
- [ ] I don't think this PR needs any additional reviewers

By adding a +1 you are confirming you have...
- Witnessed the work behaving as expected (this could be on the author's machine or screencast).
- Checked all the work locally on pattern library.

---

### CODE REVIEW

**Comment by circleBuildAgent** [comment]
<h3>Brand Version Checker</h3>
  <h4>❌ Warning: | Current Globals version: 2.2.7 ❌</h4>
  <p><i>Holiday Extras</i> is not using the latest Globals version ( <strong>^2.2.8</strong> )</p><h3>Application Checker</h3><p><i>Tripapp</i> is on version <strong>^2.1.8</strong></p><p><i>SEO Landing Pages</i> is on version <strong>2.1.6</strong></p><p><i>Manage My Booking</i> is on version <strong>~1.1.2</strong></p><p><i>Email Landing Pages</i> is on version <strong>2.1.6</strong></p><p><small><strong>Please consider these versions before merging</strong></small></p>

**Comment by circleBuildAgent** [comment]
<h3>Brand Version Checker</h3>
  <h4>❌ Warning: | Current Globals version: 2.2.8 ❌</h4>
  <p><i>Alton Towers</i> is not using the latest Globals version ( <strong>^2.2.7</strong> )</p><p><i>Chessington</i> is not using the latest Globals version ( <strong>^2.2.7</strong> )</p><p><i>Harry Potter</i> is not using the latest Globals version ( <strong>^2.2.7</strong> )</p><p><i>Legoland</i> is not using the latest Globals version ( <strong>^2.2.7</strong> )</p><p><i>Thorpe Park</i> is not using the latest Globals version ( <strong>^2.2.7</strong> )</p><p><i>Warwick</i> is not using the latest Globals version ( <strong>^2.2.7</strong> )</p><h3>Application Checker</h3><p><i>Tripapp</i> is on version <strong>^2.1.8</strong></p><p><i>SEO Landing Pages</i> is on version <strong>2.1.6</strong></p><p><i>Manage My Booking</i> is on version <strong>~1.1.2</strong></p><p><i>Email Landing Pages</i> is on version <strong>2.1.6</strong></p><p><small><strong>Please consider these versions before merging</strong></small></p>

**Comment by nassimdonald** [comment]
As discussed, use `padding` instead of `margin` and apply `list-group-icon` to item itself, rather than parent – more flexible. 

Note: Further work has been done in a separate branch for this – yet to be PRed.

File: brands/globals/src/less/list-group.less [hunk 1]
```diff
@@ -1 +1,10 @@
 @import '../../../../bootstrap/less/list-group';
+
+.list-group-icon {
+  & > li {
+    margin-left:2em;
+    a {
+      margin-left:-1em;
```
Comment (nassimdonald) [inline brands/globals/src/less/list-group.less]: Change margins to padding and apply class to item itself – consider renaming to something like `.list-group-item-with-icon`.

**Comment by circleBuildAgent** [comment]
<h3>Brand Version Checker</h3>
  <h4>❌ Warning: | Current Globals version: 2.2.8 ❌</h4>
  <p><i>Alton Towers</i> is not using the latest Globals version ( <strong>^2.2.7</strong> )</p><p><i>Chessington</i> is not using the latest Globals version ( <strong>^2.2.7</strong> )</p><p><i>Harry Potter</i> is not using the latest Globals version ( <strong>^2.2.7</strong> )</p><p><i>Legoland</i> is not using the latest Globals version ( <strong>^2.2.7</strong> )</p><p><i>Thorpe Park</i> is not using the latest Globals version ( <strong>^2.2.7</strong> )</p><p><i>Warwick</i> is not using the latest Globals version ( <strong>^2.2.7</strong> )</p><h3>Application Checker</h3><p><i>Tripapp</i> is on version <strong>^2.1.8</strong></p><p><i>SEO Landing Pages</i> is on version <strong>2.1.6</strong></p><p><i>Manage My Booking</i> is on version <strong>~1.1.2</strong></p><p><i>Email Landing Pages</i> is on version <strong>2.1.6</strong></p><p><small><strong>Please consider these versions before merging</strong></small></p>

**Comment by h0rhay** [comment]
@nassimdonald I've addressed your comment 👍

**Comment by circleBuildAgent** [comment]
<h3>Brand Version Checker</h3>
  <h4>❌ Warning: | Current Globals version: 2.2.8 ❌</h4>
  <p><i>Alton Towers</i> is not using the latest Globals version ( <strong>^2.2.7</strong> )</p><p><i>Chessington</i> is not using the latest Globals version ( <strong>^2.2.7</strong> )</p><p><i>Harry Potter</i> is not using the latest Globals version ( <strong>^2.2.7</strong> )</p><p><i>Legoland</i> is not using the latest Globals version ( <strong>^2.2.7</strong> )</p><p><i>Thorpe Park</i> is not using the latest Globals version ( <strong>^2.2.7</strong> )</p><p><i>Warwick</i> is not using the latest Globals version ( <strong>^2.2.7</strong> )</p><h3>Application Checker</h3><p><i>Tripapp</i> is on version <strong>^2.1.8</strong></p><p><i>SEO Landing Pages</i> is on version <strong>2.1.6</strong></p><p><i>Manage My Booking</i> is on version <strong>~1.1.2</strong></p><p><i>Email Landing Pages</i> is on version <strong>2.1.6</strong></p><p><small><strong>Please consider these versions before merging</strong></small></p>

**Comment by circleBuildAgent** [comment]
<h3>Brand Version Checker</h3>
  <h4>✅ All brands are up to date! ✅</h4>
  <h3>Application Checker</h3><p><i>Tripapp</i> is on version <strong>^2.1.8</strong></p><p><i>SEO Landing Pages</i> is on version <strong>2.1.6</strong></p><p><i>Manage My Booking</i> is on version <strong>~1.1.2</strong></p><p><i>Email Landing Pages</i> is on version <strong>2.1.6</strong></p><p><small><strong>Please consider these versions before merging</strong></small></p>

**Comment by circleBuildAgent** [comment]
<h3>Brand Version Checker</h3>
  <h4>✅ All brands are up to date! ✅</h4>
  <h3>Application Checker</h3><p><i>Tripapp</i> is on version <strong>^2.1.8</strong></p><p><i>SEO Landing Pages</i> is on version <strong>2.1.6</strong></p><p><i>Manage My Booking</i> is on version <strong>~1.1.2</strong></p><p><i>Email Landing Pages</i> is on version <strong>2.1.6</strong></p><p><small><strong>Please consider these versions before merging</strong></small></p>

**Comment by circleBuildAgent** [comment]
<h3>Brand Version Checker</h3>
  <h4>✅ All brands are up to date! ✅</h4>
  <h3>Application Checker</h3><p><i>Tripapp</i> is on version <strong>^2.1.8</strong></p><p><i>SEO Landing Pages</i> is on version <strong>2.1.6</strong></p><p><i>Manage My Booking</i> is on version <strong>~1.1.2</strong></p><p><i>Email Landing Pages</i> is on version <strong>2.1.6</strong></p><p><small><strong>Please consider these versions before merging</strong></small></p>

**Pull request merged**